apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.ts

```diff
@@ -291,6 +295,12 @@ export class SalesRegisterExpandComponent implements OnInit, OnDestroy {
                 "checked": true
             }
         ];
+
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
```

Comment:
use component store

Reply:
Sir, here no component store is implemented that's why I use global store.